packages/preprocess/index.cjs

```diff
@@ -159,6 +178,8 @@ const updateExtractedQueriesDir = function(content, filename){
 
     visit(tree, 'code', function(node) {
         let id = node.lang ?? 'untitled'
+         // Prevent "real" code blocks from being interpreted as queries
+         if (prismLangs.has(id.toLowerCase())) return
```

Comment:
The only trade-off I see here is if someone wants to name their query `Shell` it will not run it.

Reply:
Yes this is true. These names are now "illegal"